views/js/qtiCreator/widgets/interactions/graphicGapMatchInteraction/Widget.js

```diff
@@ -115,6 +116,123 @@ define([
             _.forEach(interaction.gapImgs, function(gapImg){
                 $gapList.append(gapImg.render());
             });
+        },
+
+        /**
+         * Display warning message in case matchMax is set to 0 (infinite) and pair is higher that 0
+         * @param {String} template
+         * @param {Object} choice
+         */
+        infinityMatchMax (template, choice) {
+            const interaction = this.element;
+            const response = interaction.getResponseDeclaration();
+
+            // if no mapEntries, there is no need to check
+            if (!response.mapEntries || !_.size(response.mapEntries)) {
```

Comment:
not right, could be possible pair and mapped default > 0

Reply:
true